linode/firewall/framework_resource_test.go

```diff
@@ -420,40 +424,6 @@ func TestAccLinodeFirewall_externalDelete(t *testing.T) {
 	})
 }
 
-func TestAccLinodeFirewall_emptyIPv6(t *testing.T) {
```

Comment:
Just curious, why're we removing this test case?

Reply:
An empty list of IP is not allowed by the API. You can try this sample call:
```bash
curl --request POST \
     --url https://api.linode.com/v4/networking/firewalls \
     --header 'accept: application/json' \
     --header "authorization: Bearer $TOKEN" \
     --header 'content-type: application/json' \
     --data '
{
  "rules": {
    "inbound_policy": "DROP",
    "outbound_policy": "ACCEPT",
    "inbound": [
      {
        "addresses": {
          "ipv4": [
            "74.102.235.249/32"
          ],
          "ipv6": []
        },
        "action": "DROP",
        "protocol": "UDP"
      }
    ]
  },
  "label": "myfw123"
}
'
```

so I added validators to prevent any empty list input on the `ipv6` and `ipv4` attributes and retired this test case.

Reply:
Actually, I can bring this test case back, but the `ipv6` attribute will be changed from `[]` to a `null` lol

Reply:
Sounds good!